docs/guides/js-waku/debug-waku-dapp.md

```diff
@@ -0,0 +1,99 @@
+---
+title: How to Debug Your Waku DApp and WebSocket
+---
+
+This guide provides detailed steps to enable and use debug logs to troubleshoot your Waku DApp, whether in a NodeJS or browser environment and check your WebSocket connections in [nwaku](/guides/run-nwaku-node).
+
+## Enabling Debug Logs
+
+When resolving issues in your Waku DApp, debug logs can be helpful. The `@waku/sdk` and `libp2p` packages use the debug tool to handle and show logs that help you debug effectively.
+
+### NodeJS Environments
+
+To enable debug logs for `@waku/sdk` on NodeJS, you must set the `DEBUG` environment variable. To only enable debug logs for `@waku/sdk`:
+
+```shell
+export DEBUG=waku*
+```
+
+To enable debug logs for both `@waku/sdk` and `libp2p`:
+
+```shell
+export DEBUG=waku*,libp2p*
+```
+
+To enable debug logs for all components:
+
+```shell
+export DEBUG=*
+```
+
+### Browser Environments
+
+To view debug logs in your browser's console, modify the local storage and add the `debug` key. Here are guides for various modern browsers:
```

Comment:
actually @danisharora099 pointed out that you can just run a command in your console `window.storage.debug="waku*"` I believe.